### PR TITLE
add context for PR description template text

### PR DESCRIPTION
**Pull request by mtbc**
See https://github.com/openmicroscopy/openmicroscopy/pull/4628#issuecomment-216797145 for context. Staged at https://www.openmicroscopy.org/site/support/contributing-staging/using-git.html#pull-requests.

### CODE REVIEW

File: contributing/using-git.txt [hunk 1]
```diff
@@ -842,6 +842,13 @@ Pull Request.
     git push gh new_stuff
     # Now go to the new_stuff branch on github.com and open the PR
 
+GitHub's "Open a pull request" page invites you to leave a comment under
```
Comment (ximenesuk) [inline contributing/using-git.txt:845]: Is the 'comment' here distinct from the description?

File: contributing/using-git.txt [hunk 2]
```diff
@@ -842,6 +842,13 @@ Pull Request.
...
+the PR title. A good choice of PR title and description are both helpful
```
Comment (ximenesuk) [inline contributing/using-git.txt]: I wonder if it is worth mentioning something about GitHub creating a title from the branch name (or commit message if the PR is just one commit) but that a more helpful title might be considered?

File: contributing/using-git.txt (hunk 1 above)
Comment (mtbc) [inline contributing/using-git.txt:845]: It's actually called a comment on the page. So, no.

File: contributing/using-git.txt (hunk 2 above)
Comment (mtbc) [inline contributing/using-git.txt]: I think probably not but if you much think otherwise happy to add!

File: contributing/using-git.txt [hunk 3]
```diff
@@ -842,6 +842,13 @@ Pull Request.
     git push gh new_stuff
     # Now go to the new_stuff branch on github.com and open the PR
 
+GitHub's "Open a pull request" page invites you to leave a comment under
+the PR title: we use this comment to describe the PR. A good choice of
+PR title and description are both helpful to reviewers of your work. If
+there is template text already provided in the comment box for you to
```
Comment (hflynn) [inline contributing/using-git.txt]: This last sentence is very wordy. Maybe:
"If there is template text already provided, do consider what it says but also feel free to edit it as necessary to make sense for your PR." ?

**Comment by ximenesuk** [comment]
This looks good to merge (assuming the code PR is merged).

**Pull request merged**